crates/uv/src/commands/pip/tree.rs

```diff
@@ -113,10 +115,14 @@ fn filtered_requirements<'env>(
 struct DisplayDependencyGraph<'env> {
     // Installed packages.
     site_packages: &'env SitePackages,
+    /// Map from package name to the installed distribution.
+    distributions: HashMap<&'env PackageName, &'env InstalledDist>,
```

Comment:
You can use `SitePackages::get_packages` instead of recreating that map here.

Reply:
hmm, is there a reason that `get_packages` returns a vector? specifically, are there times when we expect a package name to map to multiple packages? (i thought there can never be two versions of the same package, or at least that's not supposed to happen)

Reply:
It is technically possible, though not expected in any real word usage. See: https://github.com/jaraco/pip-run?tab=readme-ov-file#experiments-and-testing.